hyperjumptech/monika
# Fix: #384 chain request header is not set on chaining request

# Monika Pull Request (PR)  
Fix #384 Header value is not changed

## What feature/issue does this PR add  
Fix headers value not set on second requests on chain-request process.

## How did you implement / how did you fix it  
set directly the replaced value using Handlebars to requestConfig headers object

## How to test  
1. set chain-request config
2. set header of second request value from first request response
3. run monika using the chain request config
4. second request should send the right header value and return 200 response status

## image 
![Screen Shot 2021-09-07 at 16 58 09](https://user-images.githubusercontent.com/8952493/132326748-21fc2750-507d-46b0-87a6-7c2a64a50965.png)
